QuantLib.vcxproj

```diff
@@ -2537,6 +2538,7 @@
     <ClCompile Include="ql\pricingengines\vanilla\analyticdigitalamericanengine.cpp" />
     <ClCompile Include="ql\pricingengines\vanilla\analyticdividendeuropeanengine.cpp" />
     <ClCompile Include="ql\pricingengines\vanilla\analyticeuropeanengine.cpp" />
+    <ClInclude Include="ql\pricingengines\vanilla\analyticeuropeanvasicekengine.cpp" />

```

Comment:
This should be `ClCompile` like the rows around it.

Reply:
Have made the change and build succeeds now. Thanks.